### PR TITLE
Update cheatsheet.md

### PR DESCRIPTION
**Pull request by nirmalg-1454**
Added .vimrc settings for bash shell

### CODE REVIEW

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -32,6 +32,13 @@ You can also use a shorthand alias for `kubectl` that also works with completion
 alias k=kubectl
 complete -o default -F __start_kubectl k
 ```
+Also you can setup the VIM editor. Add below lines in ~/.vimrc
+
+```bash
+set expandtab  ### use spaces for tab
+set tabstop=2  ### amount of spaces used for tab
+set shiftwidth=2 ### amount of spaces used during indentation
+```
```
Comment (tengqm) [inline content/en/docs/reference/kubectl/cheatsheet.md:41]: This seems irrelevant to k8s.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Hey @nirmalg-1454, firstly, please sign the CLA. Secondly, I agree with @tengqm that the proposed content seems irrelevant to Kubernetes. Could you please briefly explain why the suggested content would add value to the documentation?

File: content/en/docs/reference/kubectl/cheatsheet.md (hunk 1 above)
Comment (nirmalg-1454) [reply content/en/docs/reference/kubectl/cheatsheet.md:41]: when we are setting the alias and other exports in .bashrc, then setting the .vimrc is also relevant for some users where they can set the tab as spaces while editing the yaml files. Also spaces for each tab varies from one user to another. So setting it to 2 spaces will make easy for developers to write the yaml files.

**Comment by nirmalg-1454** [comment]
> Hey @nirmalg-1454, firstly, please sign the CLA. Secondly, I agree with @tengqm that the proposed content seems irrelevant to Kubernetes. Could you please briefly explain why the suggested content would add value to the documentation?

when we are setting the alias and other exports in .bashrc, then setting the .vimrc is also relevant for some users where they can set the tab as spaces while editing the yaml files. Also spaces for each tab varies from one user to another. So setting it to 2 spaces will make easy for developers to write the yaml files.

**Comment by divya-mohan0209** [comment]
@nirmalg-1454: Honestly, what you've suggested is a matter of personal preference, and IMO, it doesn't add value to the content. However, I'll defer to the PR wrangler for the week, @natalisucks, for her review on this.

**Comment by natalisucks** [comment]
I agree with the other leads in this thread that this change doesn't improve Kubernetes documentation for our readers and contributors. @nirmalg-1454 I appreciate that this is your first contribution, but I'd suggest looking at the range of [issues](https://github.com/kubernetes/website/issues) we have as another avenue of finding a meaningful contribution to make. You can also search via the tag `good-first-issue` as a way to filter the issues.

/close
